.github/workflows/refapp-2x-vitals-and-triaging.yml

```diff
@@ -47,5 +47,5 @@ jobs:
           echo "cucumber.publish.enabled=true" > src/test/resources/cucumber.properties
           mvn clean install -DskipTests=true
           cp -f src/test/resources/org/openmrs/uitestframework/test-local.properties src/test/resources/org/openmrs/uitestframework/test.properties
-          npm run refapp2VitalsAndTriagingTest
```

Comment:
i would leave it the way it was as refapp2vitalsAndTriagingTest. it sounds more better.